x/hard/keeper/borrow.go

```diff
@@ -22,12 +22,6 @@ func (k Keeper) Borrow(ctx sdk.Context, borrower sdk.AccAddress, coins sdk.Coins
 		}
 	}
 
-	// Get current stored LTV based on stored borrows/deposits
-	prevLtv, err := k.GetStoreLTV(ctx, borrower)
-	if err != nil {
-		return err
-	}
-
 	// Call incentive hook
 	existingBorrow, hasExistingBorrow := k.GetBorrow(ctx, borrower)
 	if hasExistingBorrow {
```

Comment:
Given that we are updating the deposit interest, I think we should call `BeforeDepositModified` as well

Reply:
Fixed in https://github.com/Kava-Labs/kava/pull/800/commits/0d9bcf911e3cf6f625c7e72b40023d811679b401